protocol/chat.go

```diff
@@ -93,7 +94,9 @@ func (c *Chat) PublicKey() (*ecdsa.PublicKey, error) {
 }
 
 func (c *Chat) Public() bool {
-	return c.ChatType == ChatTypePublic
+	// It looks like status-react created profile chats as public chats
+	// so for now we need to check for the presence of "@" in their chatID
+	return c.ChatType == ChatTypePublic && !strings.HasPrefix(c.ID, "@")
```

Comment:
im not sure this will work, because updates should still be treated as public chats , so probbaly better to move this condition outside where we drop images

Reply:
good point, pushed a fix, thanks